File: pset6/mario.py
from cs50 import get_int

while True:
    height = get_int("height:")
    if height > 0 and height < 9:
        break


for i in range(1, height + 1):
    for j in range(height):
        if j > height - i - 1:
            print("#", end="")
        else:
            print(" ", end="")

    print(" " * 2, end="")

    for l in range(i):
        print("#", end="")
    print()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("height:");
    }
    while (height <= 0 || height >= 9);

    // print height
    for (int i = 1; i <= height; i++)
    {
        // print width first blocks
        for (int j = 1; j <= height; j++)
        {
            if (j > height - i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        // print two empty spaces
        for (int k = 0; k < 2; k++)
        {
            printf(" ");
        }

        // print the last blocks
        for (int l = 1; l <= i; l++)
        {
            printf("#");
        }

        printf("\n");
    }
}